packages/server/src/fhir/operations/expand.ts

```diff
@@ -397,10 +403,22 @@ export function expansionQuery(
     for (const condition of include.filter) {
       switch (condition.op) {
         case 'is-a':
-          query = addDescendants(query, codeSystem, condition.value);
-          break;
         case 'descendent-of':
-          query = addDescendants(query, codeSystem, condition.value).where('code', '!=', condition.value);
+          if (params?.filter) {
+            const base = new SelectQuery('Coding', undefined, 'origin')
+              .column('id')
+              .column('code')
+              .column('display')
+              .where('system', '=', codeSystem.id)
+              .where(new Column('origin', 'code'), '=', 'code');
```

Comment:
Is the `alias` required?  Would `new Column('Coding', 'code')` have worked?

Reply:
My bad, this was a bug.  The updated implementation should be a major improvement over the current typeahead performance